assets/prototype/application/utilities/text/entityFieldSearch/index.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { trim } from 'ramda';
+
+type EntityFieldSearchProps = {
+	entities: any[];
```

Comment:
Could this be `Entity[]` from `/domain/eventEditor/data/types.ts` instead of `any[]`?

Reply:
Perfect suggestion :100: